ops/services/postgres_db/_var.tf

```diff
@@ -49,4 +49,10 @@ variable "tls_enabled" {
 
 variable "public_access" {
   default = true
-}
\ No newline at end of file
+}
+
+variable "nophi_user_password" {
```

Comment:
Does this actually need to be an input at all? It seems like it's used to create the user and then stashed in the vault, and we're going to have metabase fetch it out of the vault at runtime, so as it stands I'm not sure I'm seeing a need for this to exist outside of `postgres_db` itself—is there a future use case I'm missing? Or would that result in the password being changed every time the module ran, or something similarly unfortunate?

Reply:
Sharp eye. Normally no, this could be entirely self-contained in the DB module. The problem here is that there's no way to declare a variable sensitive unless it's coming from some other external source (`var`, `outputs`, etc.). Nor is there a way to suppress output from the TF statement itself. So, when we go to actually create the user, we get to see the SQL including the `WITH PASSWORD` in plain text, which is bad. Moving this to a variable lets us mark this as sensitive at the cost of even more copy-pasta.

Reply:
Wow. The More You Know—thanks!